src/InfoBubble/InfoBubble.js

```diff
@@ -0,0 +1,86 @@
+import { ClickAwayListener } from '@mui/material';
+import Tooltip from '@mui/material/Tooltip';
+import { dcnb } from 'cnbuilder';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Button } from '../Button/Button';
+import { alignOptions } from './InfoBubble.levers';
+
+export const InfoBubble = ({
+  alignment,
+  className,
+  children,
+  text,
+  ...props
+}) => {
+  const [open, setOpen] = useState(false);
+
+  const handleTooltipClose = () => {
+    setOpen(false);
+  };
+
+  const handleTooltipOpen = () => {
+    setOpen(true);
+  };
+
+  const bubbleStyles = {
+    tooltip: dcnb(
+      className,
+      '!su-px-18 !su-pb-18 !su-pt-12 !su-sans !su-text-14 !su-text-black !su-bg-black-10 !su-color-black-10 !su-shadow !su-max-w-[25ch]'
+    ),
+    tooltipArrow: '!su-bg-black-10 !su-color-black-10',
+  };
+
+  return (
+    <ClickAwayListener onClickAway={handleTooltipClose}>
+      <Tooltip
+        {...props}
+        PopperProps={{
+          disablePortal: true,
+        }}
+        onClose={handleTooltipClose}
+        open={open}
+        disableFocusListener
+        disableHoverListener
+        disableTouchListener
+        title={text}
+        placement={alignment}
```

Comment:
For some reason, I couldn't seem to get the levers (e.g. `const placement = alignOptions[alignment];` to work without encountering an error. For now I have the alignment props passed straight into placement and the alignOptions controls available for the user to select their preferred alignment in the Storybook UI. 

Do note that the MUI Tooltip is preconfigured so that it will not cut outside of the screen (e.g. if you set the alignment to `top`, it will display the Tooltip on the `bottom`. Cool stuff 😎

Reply:
I believe mui does a lot of element level styling in addition to generating style classes that have more specificity than a single class selector (tbh, most react component libs do) so we're likely going to run up against this a lot when trying to apply tailwind classes 🦐

Reply:
> I believe mui does a lot of element level styling in addition to generating style classes that have more specificity than a single class selector (tbh, most react component libs do) so we're likely going to run up against this a lot when trying to apply tailwind classes 🦐

That was supposed to be a `:shrug:`, but it autocompleted to `:shrimp:` and so there it is.

Reply:
> > I believe mui does a lot of element level styling in addition to generating style classes that have more specificity than a single class selector (tbh, most react component libs do) so we're likely going to run up against this a lot when trying to apply tailwind classes 🦐
> 
> That was supposed to be a `:shrug:`, but it autocompleted to `:shrimp:` and so there it is.

Also this was in reference to the [other comment you made](https://github.com/SU-SWS/decanter-react/pull/79#discussion_r720544853). It's definitely the end of the week... 🤦🏼

Reply:
I was wondering about the 🦐 🤷🏼‍♀️ 😂 

And on the styling/!important TW classes topic:

So I haven't looked into the Unstyled MUI components. Are all components available with the Unstyled version? I wonder what's the effort is to style the Unstyled (if it's available) vs overriding the Styled ones.

Reply:
I _believe_ that mui unfortunately only offers a subset of their components in unstyled versions and that `Tooltip` is not in that subet. Is that correct @rebeccahongsf ?

Reply:
That is correct @mattanglin! It seems like only the Autocomplete and Modal/Dialog are available unstyled in @mui/core ☹️

Reply:
Yeah....I suspected that's the case. A bit unfortunate but 🤷🏼‍♀️